strings/en_US/messages.po

```diff
@@ -4007,3 +4007,6 @@ msgstr "Your Dojo must ve verified in order to use that functionnality"
 
 msgid "Congratulations on creating your event. When your Dojo is verified by the CoderDojo team, it will appear on your public listing."
 msgstr "Congratulations on creating your event. When your Dojo is verified by the CoderDojo team, it will appear on your public listing."
+
+msgid "Email is already associated with an account. Use a new email or leave it blank."
```

Comment:
I think there should be a mention of how if you leave it blank, you will receive all correspondence for your child.

Thoughts @conoramurphy?

Reply:
Ya good point. >>

Use a new email or leave it blank to receive all correspondence on their behalf.